### DISCUSSION TITLE
Assembleia Geral da APyB

### DISCUSSION THREAD

**Discussion by tiidadavena** [apyb/comunidade#94] [Divulgação]
A Associação Python Brasil vem convidar associadas/os/es a participar de sua Assembleia Geral que acontecerá durante a décima nona edição do encontro brasileiro da comunidade Python, a Python Brasil 2023 (https://buff.ly/3RH4a18).

A assembleia deste ano visa apresentar e discutir os seguintes tópicos:

1. Eleição da cidade-sede da Python Brasil 2025;
2. Votação da nova diretoria para a gestão 2024/2025;
3. Prestação de contas da APyB;
4. Novas demandas da comunidade (confira o Github Discussions para adicionar itens da pauta a serem debatidos: https://buff.ly/3LuZDey ).

A edição deste ano terá o formato hiíbrido, com a versão online a ser realizada no servidor do Discord da Python Brasil 2023 e com a versão presencial a ser realizada no endereço a seguir:

Faculdade Anhanguera - Campus Caxias do Sul
Av. Alexandre Rizzo, 505 - Sala 01 - Desvio Rizzo, Caxias do Sul - RS - 95110-000

![1](https://github.com/apyb/comunidade/assets/12106738/90ca6acc-ddf7-4cc4-9b44-9d05b3ff7e43)
![2](https://github.com/apyb/comunidade/assets/12106738/53a02e46-9577-4e95-8788-f5b28132ee77)

A Assembleia Geral ocorrerá no dia 04/11/2023 das 13h30 até 16h.

Contamos com a sua presença!

👋 https://linktr.ee/apyb

**Comment by devmessias** [comment]
Oi @tiidadavena , estou tentando buscar informações sobre  como candidatar a minha cidade e onde eu encontro material. No twitter me falaram para eu me associar a python Brasil, o que já fiz, mas agora tenho outras dúvidas:

1 - Existe algum material/tempalte que eu possa usar para preparar a candidatura da minha cidade. No caso Uberlândia-MG
2 - Tenho que preencher algum formulário?
3 - Onde eu consigo ver as propostas das outras cidades dos anos anteriores? E de 2025?
4 -Existe algum documento com regras sobre a candidatura?
5 - Existe algum material de marketing da python Brasil para compartilhar com possíveis patrocinadores? Número médio de participantes no evento, cidades que já passou etc. Vou conversar com um pessoal aqui da minha região e estou montando uma apresentação sobre.

**Reply by tiidadavena** [reply]
Olá, @devmessias!

Respondemos suas dúvidas via email.

**Reply by tiidadavena** [reply]
Deixando as respostas públicas para caso mais pessoas compartilhem da mesma dúvida:

----

1 - A candidatura é sempre para dois anos após o evento, no caso 2025?
**Sim, tendo em vista de que é um evento nacional, requer mais preparo da Organização, recrutar pessoas organizadoras e voluntárias e também a organização do ano seguinte participa como co-organizadora do ano presente (no caso da PyBR 2024, a org. de PyBR 2025 participará para acompanhar e aprender como funcionam os fluxos).**

2 - Existe algum material/tempalte que eu possa usar para preparar a candidatura da minha cidade. No caso Uberlândia-MG
**Um template oficial não temos, mas temos modelos de candidaturas anteriores que podes dar uma olhada para embasar a candidatura da sua cidade: https://manual.pythonbrasil.org.br/proposta/introducao.html**

3 - Tenho que preencher algum formulário?
**Sim, e em breve divulgaremos o link (divulgaremos no grupo de Associados, nas nossas redes sociais e no Github Discussions).**

4 - Onde eu consigo ver as propostas das outras cidades dos anos anteriores? E de 2025?
**Conforme respondemos na pergunta 2, o link acima permite ver alguns modelos. O preenchimento do form visa dar uma base para a candidatura, mas durante a Assembleia Geral 2023, as cidades podem fazer uma breve apresentação (no máximo 10min). Sobre a apresentação da candidatura da cidade-sede, podem ser apresentados slides, docs ou apenas de forma verbal: mas lembrem-se de que vocês passarão por votação e então é interessante valorizar a sua candidatura para receber os votos.**

Esperamos que tenhamos conseguido esclarecer as suas dúvidas e estamos à disposição para esclarecer quaisquer dúvidas.

**Reply by tiidadavena** [reply]
cc @devmessias

**Reply by tiidadavena** [reply]
@devmessias o Formulário para inscrever a sua cidade está disponível neste tópico: https://github.com/apyb/comunidade/discussions/95

**Comment by gabubellon** [comment]
@tiidadavena e @apyb/diretoria 
Primeiro obrigado pela chamada da assembleia.

Quero deixar alguns pontos

* Séria legal divulgar esse site https://associacao.python.org.br/members/list/ para que quem é associado verifique o status e possa deixa a cota de associado OK para a assembleia.

* Acho que séria legal fazer um (ou mais) post de campanhas para termos nossas associações até a assembleia

* >  servidor do Discord da Python Brasil 2023

Seria legal enviar um email/convite a todos os associados alguns dias antes (talvez com um pequeno tutorial) de como será participar e votar na assembleia de maneira virtual (contém comigo para ajudar nisso ❤️ )

* Acho que seria legal fazer uma "chamamento" de candidatos ( Diretoria , Conselhos e Cidade) talvez 1 semana antes da Assembleia pra se ter tempo que as pessoas sejam conhecidas e apresentadas para a comunidade esse ano, visto o modelo híbrido.

**Reply by tiidadavena** [reply]
Agradecemos os feedbacks e as demandas! Sim, estamos em planejamento para divulgar as informações sobre as Eleição Diretoria 2024/2025, cidade-sede e lembrete para associação.

Sobre o link do Discord, atualmente estamos ajustando o servidor da Python Brasil 2023 e certamente o divulgaremos em breve.

Acerca do alerta para pagamento, a plataforma envia email automaticamente com o lembrete de vencimento da anuidade.

@rougeth a gente conseguiria implementar essa automação
 > Séria legal divulgar esse site https://associacao.python.org.br/members/list/ para que quem é associado verifique o status e possa deixa a cota de associado OK para a assembleia. ?

**Reply by rougeth** [reply]
> @rougeth a gente conseguiria implementar essa automação

Como já mandamos alguns e-mails de alerta para renovação, acho que já seria o suficiente. Se acontecer de alguém esquecer, a pessoa pode regularizar seu status até poucos minutos antes da assembleia.

@tiidadavena talvez eu tenha me perdido nas mensagens, mas era sobre isso mesmo?

**Comment by anicelysantos** [comment]
Opa! :)

Não sei se aqui é o lugar certo para isto, mas quero propor a inclusão para discussão na Assembleia, a criação de um termo de responsabilidades para lideres de organização (Big Kahuna) em eventos tanto regionais quanto o nacional.

**Reply by tiidadavena** [reply]
É sim e ficamos felizes com a sua contribuição!

Achamos o ponto extremamente relevante para levar para o debate para a Assembleia.

**Comment by jbiason** [comment]
Sugestão: oficializar a ideia de “contribuidor individual para às PythonBrasil, junto com as empresas patrocinadoras, para que pessoas possam patrocinar o evento como pessoas ao invés de empresa. 

O valor poderia ser abaixo do patrocinador bronze (algo como R$ 500, considerando valores da PythonBrasil Caxias).
